### ISSUE TITLE
Strong mode issues

### ISSUE THREAD

**Issue by har79** [dart-lang/coverage#136]
`> dartanalyzer --strong . | sort | uniq`

```
4 errors, 14 warnings and 2 hints found.
Analyzing [.]...
[error] Field declaration Resolver.failed cannot be overridden in BazelResolver. (/usr/local/google/home/hcameron/github/coverage/lib/src/resolver.dart, line 93, col 3)
[error] The argument type 'List' cannot be assigned to the parameter type 'Iterable<File>'. (/usr/local/google/home/hcameron/github/coverage/bin/format_coverage.dart, line 44, col 36)
[hint] Unused import (/usr/local/google/home/hcameron/github/coverage/lib/src/formatter.dart, line 4, col 8)
[warning] Unsound implicit cast from dynamic to List<Map<dynamic, dynamic>> (/usr/local/google/home/hcameron/github/coverage/lib/src/hitmap.dart, line 74, col 31)
[warning] Unsound implicit cast from dynamic to List<Map<dynamic, dynamic>> (/usr/local/google/home/hcameron/github/coverage/test/lcov_test.dart, line 168, col 29)
[warning] Unsound implicit cast from dynamic to List<String> (/usr/local/google/home/hcameron/github/coverage/bin/format_coverage.dart, line 186, col 18)
[warning] Unsound implicit cast from Future<dynamic> to Future<Null> (/usr/local/google/home/hcameron/github/coverage/test/test_util.dart, line 34, col 10)
[warning] Unsound implicit cast from List<dynamic> to List<Map<dynamic, dynamic>> (/usr/local/google/home/hcameron/github/coverage/test/collect_coverage_test.dart, line 62, col 31)
[warning] Unsound implicit cast from List<dynamic> to List<Map<dynamic, dynamic>> (/usr/local/google/home/hcameron/github/coverage/test/run_and_collect_test.dart, line 47, col 31)
```

**Comment by cbracken** [comment]
Thanks! Will take a look asap.

**Comment by cbracken** [comment]
Fixed by #138

**Comment by cbracken** [comment]
Published v0.8.0+2 with these fixes. Thanks again for reporting.

**Comment by har79** [comment]
Thanks for the quick response Chris!
